src/styles/bookList.css

```diff
@@ -0,0 +1,24 @@
+body {
+  background-color: #fafafa;
+}
```

Comment:
### Your Project
![Student-BookStore - Copy (2)](https://user-images.githubusercontent.com/93039359/205460846-522f6c94-4d04-4c78-8dac-3787aec0e05c.PNG)

### Template
![bookstore](https://user-images.githubusercontent.com/93039359/205460851-1d52d605-147a-4513-a32c-8eb4ccbf0f48.png)

- [ ] Kindly compare your design with the template and you will find that all your contents should be in that black box which i have highlighted 👍 to follow the template 😊